docs/source/bootstrapping.md

```diff
@@ -46,12 +47,14 @@ Options:
   -t, --toolchain-account TEXT    Account Id of the Toolchain account trusted
                                   to assume the Target account's Deployment
                                   Role  [required]
-  -b, --permissions-boundary TEXT  ARN of a Managed Policy to set as the
+  -b, --permissions-boundary TEXT
+                                  ARN of a Managed Policy to set as the
                                   Permission Boundary on the Toolchain Role
   --synth / --no-synth            Synthesize a CFN template only...do not
                                   deploy  [default: no-synth]
   --profile TEXT                  The AWS profile to initiate a session
   --region TEXT                   AWS region to use
+  --qualifier TEXT                A qualifier to append to toolchain role
```

Comment:
Are you appending qualifier only to toolchain? Not to the relevant deployment roles?

Reply:
it is being appended to toolchain role...that text is off